src/wp-includes/block-template-utils.php

```diff
@@ -810,27 +810,27 @@ function get_block_template( $id, $template_type = 'wp_template' ) {
 }
 
 /**
- * Retrieves a single unified template object using its id.
+ * Retrieves an array of template objects by id.
  *
  * @since 5.9.0
  *
  * @param string $id            Template unique identifier (example: theme_slug//template_slug).
  * @param array  $template_type Optional. Template type: `'wp_template'` or '`wp_template_part'`.
```

Comment:
Shouldn't `$template_type` be `string`? It's passed to `_get_block_template_file()` which reads:

```
@param string $template_type 'wp_template' or 'wp_template_part'.
```

If so, then I believe the same change is also needed in:
-  Filters `pre_get_block_file_template` and `get_block_file_template`.
- `get_block_template()` + Filters `pre_get_block_template` and `get_block_template`.
- `get_block_templates()` + Filters `pre_get_block_templates` and `get_block_templates`.
- `_build_block_template_result_from_file()`.

Reply:
Good catch!